public/app/plugins/datasource/cloud-monitoring/components/VisualMetricQueryEditor.test.tsx

```diff
@@ -193,22 +271,24 @@ describe('VisualMetricQueryEditor', () => {
   });
 
   it('updates labels on time range change', async () => {
-    const timeSrv = getTimeSrv();
+    replace = (target?: string) => target || '';
+    const range = getDefaultTimeRange();
     const query = createMockTimeSeriesList();
     const onChange = jest.fn();
     const datasource = createMockDatasource({
       getMetricTypes: jest.fn().mockResolvedValue([createMockMetricDescriptor()]),
-      getLabels: jest
-        .fn()
-        .mockResolvedValue(
-          timeSrv.time.from === 'now-6h' ? { 'metric.test_groupby': '' } : { 'metric.test_groupby_1': '' }
-        ),
-      templateSrv: new TemplateSrv(),
-      timeSrv,
+      getLabels: jest.fn().mockResolvedValue({ 'metric.test_groupby_1': '' }),
```

Comment:
I think the conditions of this test might be wrong now. This should be testing that the labels will change from `metric.test_groupby` to `metric.test_groupby_1`